plugins/woocommerce/src/Admin/Features/ProductDataViews/Init.php

```diff
@@ -44,34 +42,6 @@ public static function is_product_data_view_page() {
 		// phpcs:enable WordPress.Security.NonceVerification
 	}
 
-	/**
-	 * Checks for data views support.
-	 */
-	private function has_data_views_support() {
-		if ( Utils::wp_version_compare( '6.6', '>=' ) ) {
-			return true;
-		}
-
-		if ( is_plugin_active( 'gutenberg/gutenberg.php' ) ) {
```

Comment:
Wouldn't this Gutenberg check still be needed?

Reply:
Removing this should be fine as we did be on a minimum of WP 6.7.
And we were checking if WP was above 6.6 anyway and returning `true` before this check was done.

Reply:
On a side note, this code is under a experimental feature flag ( not possible to enable by users ), which is disabled by default.